<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>514</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="windowIcon">
   <iconset resource="ico.qrc">
    <normaloff>:/ToolBox.png</normaloff>:/ToolBox.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QPushButton" name="trayControl">
     <property name="text">
      <string>Tray Control</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="makeItems">
     <property name="text">
      <string>make items</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="Account" name="accountWidget" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Account</class>
   <extends>QWidget</extends>
   <header>../Account/Account.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="ico.qrc"/>
 </resources>
 <connections/>
</ui>
